### PR TITLE
fix(DB/Gameobject): Pool triple Truesilver Vein in Desolace

### PR DESCRIPTION
**Pull request by avarishd**
## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/8608
- Closes https://github.com/chromiecraft/chromiecraft/issues/1736

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [ ] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [x] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

.go ga 9256,9257,9258
Only one should be spawned at a time

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by neifion-00000000** [comment]
so the way these work is actually more complicated than this, but this would be an improvement over existing so might want to be considered regardless, or updated to share with a mithril node at a low %.

TLDR is TS/rare veins overlap other mining veins and any genuine sniffed location of a TS vein location is also that of another mine (in this specific case prolly mithril, but it can place hold with other mines as well)

**Comment by avarishd** [comment]
Possibly iron too, "judging" by wowhead. (Yes, I know)

**Comment by avarishd** [comment]
Something like that? @neifion-00000000

**Comment by neifion-00000000** [comment]
this will work I think, but you could potentially do it all as a single pool with chance if you'd like for this specific case as you are only trying to make 1 of those spawn at once right?  I do believe this is the correct method anyway, all evidence I have supports that its much more likely things are all on their own individuals pools rather than large pools with multiple spawns, basically all but proven

prolly don't need to go back and change it but so you know, the reason they do pool of pool is to allow multiple picks of pooled objects without the overlap

**Comment by avarishd** [comment]
That was useful information. Thanks!
But this will cause to sometimes (most of the time) to not have anything spawned, which I guess would be the intended behaviour?

One question though, how/where do you come up with the chances?

**Comment by neifion-00000000** [comment]
> That was useful information. Thanks! `Pool Id 11699 has no equal chance pooled entites defined and explicit chance sum is not 100. The pool will not be spawned.`
> 
> One question though, how/where do you come up with the chances?

They were your values I just split them in 3 cuz there's 3 locations?  I didn't check the math tho so I might not have added up to 100 properly and been 99.99 or something.

**Comment by avarishd** [comment]
[The error was caused because they weren't equally chanced.](https://www.azerothcore.org/wiki/pool_gameobject#chance)

**Review by Nyeriah — APPROVED** [review]
